mlflow/utils/process.py

```diff
@@ -18,8 +18,11 @@ def exec_cmd(
     throw_on_error -- if true, raises an Exception if the exit code of the program is nonzero
     env -- additional environment variables to be defined when running the child process
     cwd -- working directory for child process
-    stream_output -- if true, does not capture standard output and error; if false, captures these
-      streams and returns them
+    capture_output -- If True, captures command stdout/stderr output and when command
+                      failed, attach captured stdout/stderr output to exception message and
+                      raise exception; if False, does not capture command stdout/stderr output
+                      and when command failed, raise exception without stdout/stderr
+                      output attached.
```

Comment:
@WeichenXu123 I think I'd simplify this to the following:

```suggestion
    capture_output -- If True, stdout and stderr will be captured and included in an exception message on failure; if False, these streams won't be captured.
```